### ISSUE TITLE
Series/serial interface issue

### ISSUE THREAD

**Issue by JannTibetan** [buda-base/public-digital-library#796]
I've found that when looking at the umbrella record for a series there is a bug with the Related Resources options. The "All" link leads to a new pages that says there are no related texts, whereas the red "See More" button works correctly by taking you to all the items in the series.
See http://purl.bdrc.io/resource/WAS3CN8396
 ![image](https://user-images.githubusercontent.com/44705185/217646651-b45baa9b-4a35-4c67-8c1d-14e717a4888c.jpeg)

**Comment by berger-n** [comment]
thanks! it's fixed now: https://library.bdrc.io/show/bdr:WAS3CN8396

![image](https://user-images.githubusercontent.com/33383857/217839490-bebbc161-4eba-4a9d-bb56-a2e3c78ca554.png)

**Comment by JannTibetan** [comment]
Thanks. It's working great now.

Is it possible to post a description of the series directly under the team headline? 
![IMG_6137](https://user-images.githubusercontent.com/44705185/217867048-0e58eb31-b3e6-4f93-bb5d-6841cc524d9b.jpg)

**Comment by eroux** [comment]
yes, through the "description" property in the editor (what's currently there are notes)

**Comment by JannTibetan** [comment]
@karmagongde can you put Kelsang Lhamo's description of this series into the description field? I think it will look better if the description is more prominent. Thank you.

**Comment by karmagongde** [comment]
@JannTibetan I moved it to the description field.
